src/dragonfly_speech_recognition/dragonfly_server.py

```diff
@@ -52,7 +52,13 @@ def spin(self):
             conn.close()
 
     def _process_connection(self, conn):
-        grammar, target = conn.recv()
+        # parse command
+        cmd = conn.recv()
+        if cmd == 'restart_node':
+            logger.warn('restarting node now!!!')
+            exit(1)
```

Comment:
I do agree, but then we need to change the interface from "restart_node" to "exit_node". However this is also fine with me